filebeat/input/filestream/prospector_test.go

```diff
@@ -97,13 +101,13 @@ func TestProspector_InitCleanIfRemoved(t *testing.T) {
 }
 
 func TestProspector_InitUpdateIdentifiers(t *testing.T) {
-	f, err := ioutil.TempFile("", "existing_file")
+	f, err := os.CreateTemp("", "existing_file")
```

Comment:
nit.  I think using t.TempDir() would be preferred.  That way we know the temp file will get deleted when the test exits.

Reply:
You're right, that's one of the lint issues I fixed, so I just replaced the `ioutil` call by its equivalent.

I'll make the change you suggested.